src/features/broker-protection/comparisons/constants.js

```diff
@@ -1,89 +1,1476 @@
-    abraham: ['abraham', 'abe'],
-    alexander: ['alexander', 'alex'],
-    alexandra: ['alexandra', 'alex'],
-    alexis: ['alexis', 'lexi'],
-    anthony: ['anthony', 'tony'],
-    ben: ['ben', 'benjamin'],
-    benjamin: ['benjamin', 'ben'],
-    bev: ['bev', 'beverly'],
-    beverly: ['beverly', 'bev'],
-    catherine: ['catherine', 'cathy'],
-    cathy: ['cathy', 'catherine'],
-    charles: ['charles', 'charlie'],
-    charlie: ['charlie', 'charles'],
-    chris: ['chris', 'christopher'],
-    christopher: ['christopher', 'chris'],
-    clinton: ['clinton', 'clint'],
-    dan: ['dan', 'daniel', 'danny'],
-    daniel: ['daniel', 'dan', 'danny'],
-    danny: ['danny', 'dan', 'daniel'],
-    dave: ['dave', 'david'],
-    david: ['david', 'dave'],
-    don: ['don', 'donald'],
-    donald: ['donald', 'don'],
-    ed: ['ed', 'edward', 'eddie'],
-    eddie: ['eddie', 'ed', 'edward'],
-    edward: ['edward', 'eddie', 'ed'],
-    fred: ['fred', 'frederick', 'freddie'],
-    freddie: ['freddie', 'frederick', 'fred'],
-    frederick: ['frederick', 'fred', 'freddie'],
-    jacob: ['jacob', 'jake'],
-    jake: ['jake', 'jacob'],
-    james: ['james', 'jim'],
-    jeff: ['jeff', 'jeffery'],
-    jeffery: ['jeffery', 'jeff'],
-    jim: ['jim', 'james'],
-    jon: ['jon', 'jonathan'],
-    jonathan: ['jonathan', 'jon'],
-    josh: ['josh', 'joshua'],
-    joshua: ['joshua', 'josh'],
-    katherine: ['katherine', 'katie'],
-    katheryn: ['katheryn', 'katie'],
-    kim: ['kim', 'kimberly'],
-    kimberly: ['kimberly', 'kim'],
-    lexi: ['lexi', 'alexis'],
-    lucas: ['lucas', 'luke'],
-    luke: ['luke', 'lucas'],
-    matt: ['matt', 'matthew'],
-    matthew: ['matthew', 'matt'],
-    michael: ['michael', 'mike'],
-    mike: ['mike', 'michael'],
-    nate: ['nate', 'nathan', 'nathaniel'],
-    nathan: ['nathan', 'nathaniel', 'nate'],
-    nathaniel: ['nathaniel', 'nathan', 'nate'],
-    nicholas: ['nicholas', 'nick'],
-    nick: ['nick', 'nicholas'],
-    patricia: ['patricia', 'pat'],
-    ray: ['ray', 'raymond'],
-    raymond: ['raymond', 'ray'],
-    richard: ['richard', 'rich', 'rick'],
-    rob: ['rob', 'robert', 'bob'],
-    robert: ['robert', 'rob', 'bob'],
-    rus: ['rus', 'russell'],
-    russell: ['russell', 'rus'],
-    sam: ['sam', 'samuel', 'sammy'],
-    samuel: ['samuel', 'sam', 'sammy'],
-    stan: ['stan', 'stanley'],
-    stanley: ['stanley', 'stan'],
-    sue: ['sue', 'susan'],
-    susan: ['susan', 'sue'],
-    ted: ['ted', 'teddy'],
-    teddy: ['teddy', 'ted'],
-    thad: ['thad', 'thaddeus'],
-    thaddeus: ['thaddeus', 'thad'],
-    thomas: ['thomas', 'tom', 'tommy'],
-    tim: ['tim', 'timothy'],
-    timothy: ['timothy', 'tim'],
-    tom: ['tom', 'thomas', 'tommy'],
-    tommy: ['tommy', 'tom', 'thomas'],
-    tony: ['tony', 'anthony'],
-    will: ['will', 'william'],
-    william: ['william', 'will'],
-    zach: ['zach', 'zachary'],
-    zachary: ['zachary', 'zach']
+    aaron: ['erin', 'ronnie', 'ron'],
```

Comment:
this list is huge now (which is great!) but we might want to defer the evaluation until we know it's needed. For example, after bundling, this particular module might get hoisted into the first execution context, so things like duck player (living in the same bundle) would be incurring the memory cost of loading this.

Reply:
Yep, that's a good idea. Will split this list out and dynamically import.

Reply:
it will still have to be in the same bundle (we can't load files dynamically) - it's more about the variable that holds this list being at the top-level of the module/file that it's in. that could cause it to be evaluated for everyone. I can check this before you make the change

Reply:
Ah, ok. Yeah, that'd be great. Thank you.